qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/VisitNotePage.java

```diff
@@ -119,10 +120,12 @@ public void confirmDeleteButton() {
 	}
 
 	public String primaryDiagnosis() {
+		waiter.until(ExpectedConditions.visibilityOfElementLocated(PRIMARY_DIAGNOSIS_ELEMENT));
```

Comment:
What value does the above line add?

Reply:
Basically, this was to help us confirm that using this method anywhere will return error with waiting visibility issue when we directly use it to check for an assertion.

Reply:
Isn't already done by `findElement`?

Reply:
True dat.